estate/models/EstateProperty.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from odoo import models, fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "The estate_property table stores real estate property details and metadata for transaction management."
+    active = fields.Boolean(default=False)
+    property_name = fields.Char(required=True, tracking=True, default="Unknown") 
+
+    create_uid = fields.Many2one('res.users', string="Created by", readonly=True, ondelete="set null")
+    create_date = fields.Datetime(string="Creation Date", readonly=True)
+    write_uid = fields.Many2one('res.users', string="Updated by", readonly=True, ondelete="set null")
+    write_date = fields.Datetime(string="Last Updated", readonly=True)
+    last_seen = fields.Datetime(string="Last Seen", readonly=True, default=datetime.now())
+
+
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(default= (datetime.today()))
```

Comment:
```suggestion
    date_availability = fields.Date(default=datetime.today())
```
Bad extra brackets reducing code readability, also can we only pass a value here as a default value? What can be other possibility?